### DISCUSSION TITLE
Does anyone know of a user who owns the "Open Sourcer" achievement?

### DISCUSSION THREAD

**Discussion by S1i** [drknzz/GitHub-Achievements#64] [Q&A] [answered]
I'd like to see the open sourcer achievement in action and on someone's profile so I'm wondering if anyone knows of a user who has it?

p.s, marking this answer for someone who doesn't have the galaxy brain badge

**Comment by AngeloAntonioBottaro** [comment, marked as answer]
I don't

**Comment by DanielR-spec** [comment]
It's interesting question
